var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -423,10 +427,14 @@ def cmake_args(self):
             self.define_from_variant("QE_ENABLE_MPI_GPU_AWARE", "mpigpu"),
         ]
 
-        if "+gipaw" in spec:
-            cmake_args.append(self.define("QE_ENABLE_PLUGINS", "gipaw"))
+        plugins = []
+
+        if "+fox" in spec:
```

Comment:
At some point you might want to start transitioning to `spec.satisfies()` for variant checks, especially if the variant is not unique to the current package.

From the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints:

"The difference between the two functions is that `satisfies()` tests whether spec constraints overlap at all, while `in` tests whether a spec _or any of its dependencies_ satisfy the provided spec."